.github/workflows/update_structured_experiments_readme.yml

```diff
@@ -0,0 +1,39 @@
+# Unless specified, the working directory is the root of the repo
+# This is the name displayed on GitHub under the actions tab
+name: auto-update
+
+# Defines what repository event triggers the action
+on:
+  # Triggers the workflow on push
+  push:
+    # Restricts the trigger to the structured experiments directory and sub-directories
+    paths:
+      - 'structured_experiments/**'
+    # Only triggers when push is to the main branch
+    branches: [ main ]
+
+# Defines an individual job
+jobs:
+  # Operating system the runner is built with
+  build:
+    runs-on: ubuntu-latest
+    # Defines each step of this job
+    steps:
+      # Checks out repo under the $GITHUB_WORKSPACE so the workflow has access
+      - uses: actions/checkout@v2
+      # Executes the python file to modify the readme
+      - name: update structured experiments readme
+        run: python ./.github/workflows/update_structured_experiments_readme.py ./structured_experiments

```

Comment:
This line is essentially just a cli command which runs the python file with the path to the structured experiments folder as an argument.

If you were to run python <file> <argument> locally, the argument would be changed based on the context in which the python file is run.

If you run the file directly in the structured_experimenta folder, the argument passed would be ./ and if you run it from the root of the repo it would be what you see.

Does that answer the change request or am I misunderstanding your question?

Reply:
Resolving based on external conversation.